go/carmen/carmen.go

```diff
@@ -198,6 +200,11 @@ type HistoricBlockContext interface {
 	// otherwise, the proof is returned.
 	GetProof(address Address, keys ...Key) (WitnessProof, error)
 
+	// CreateLiveDBGenesis writes a LiveDB genesis file for given block into out.
+	// The genesis file is exported from the archive for the
+	// given block and is used to sync LiveDB to that block.
+	CreateLiveDBGenesis(out io.Writer) (Hash, error)
```

Comment:
I think the term `Genesis` should not be in this function name. While it is true that what gets exported is potentially part of a genesis file, it is only a small part of it, and hardly the defining element.

I would suggest to rename this function to something like`Export()`. Wdyt?

Reply:
You are right, this is only part of the whole `genesis` file. I will remove the term from the whole PR and simply use `Export()`.